alchemiscale/compute/api.py

```diff
@@ -9,7 +9,8 @@
 import json
 from datetime import datetime, timedelta
 
-from fastapi import FastAPI, APIRouter, Body, Depends, HTTPException, status
+from fastapi import FastAPI, APIRouter, Body, Depends, HTTPException
+from fastapi import status as http_status
```

Comment:
Currently, `compute.api` doesn't use `HTTPException` or `status`. We can just remove them.

Reply:
We should start using them in various places to give more descriptive error messages, so I'd rather leave them in as a kind of placeholder.

Reply:
Accepted your suggestion; can proceed later with adding these back in as needed.